src/Telemetry/Metering/Metrics.Generators/Model/RequestInfo.cs

```diff
@@ -7,13 +7,20 @@
 namespace Metrics.Generators;
 
 // The structure that contains the metering information about the request.
+// XML docs for type's members are needed to generate the metrics report.
 internal struct RequestInfo
 {
     // This annotated property will be used as a tag for the RequestStats histogram.
```

Comment:
Change this to a `<remarks />`?

Reply:
I didn't want these comments to be treated as XML docs - so that it won't get to the report